src/sentry/migrations/0001_squashed_0200_release_indices.py

```diff
@@ -3646,9 +3647,7 @@ class Migration(migrations.Migration):
                 ("email", models.EmailField(max_length=75)),
                 (
                     "validation_hash",
-                    models.CharField(
-                        default=sentry.models.useremail.default_validation_hash, max_length=32
-                    ),
+                    models.CharField(default=sentry.utils.security.get_secure_token, max_length=32),
```

Comment:
This is triggering the migrations required GitHub action. Should I add back a `default_validation_hash()` or can we force merge it like this?

Reply:
I think this is a reasonable change to make.